lib/veryfi/request.rb

```diff
@@ -102,9 +103,7 @@ def generate_signature(params, timestamp)
       Veryfi::Signature.new(client_secret, params, timestamp).to_base64
     end
 
-    def process_response(_http_verb, response)
-      # return response if http_verb == :delete
-
+    def process_response(response)
       JSON.parse(response.body)
```

Comment:
Does it account for empty body? Sometimes back-end can just return an empty message.